src/social-network/defaults/worker.ts

```diff
@@ -0,0 +1,8 @@
+import { nop } from '../../utils/utils'
+import { cloneDeep } from 'lodash-es'
+
+export const defaultSocialNetworkWorker = cloneDeep({
+    autoVerifyBio: nop,
```

Comment:
Do not provide a default value for them. It will effect how UI display how many options user can use.

Reply:
1. c7a01283
2. 52dfbbd7 use null instead of undefined